Misp v3 delete attribute + local tag arg (#17742)
demisto/content: Packs/MISP/ReleaseNotes/2_1_0.md

#### Integrations
##### MISP v3

> - Added the command ***misp-delete-attribute*** which allow users to delete an attribute according to the given attribute ID.
> - In the **misp-add-tag-to-event** command the *is_local* argument was added.
> - In the **misp-add-tag-to-attribute** command the *is_local* argument was added.

Reviewer: ```suggestion
- Added the command ***misp-delete-attribute*** which allows users to delete an attribute according to a given attribute ID.
- Added the *is_local* argument to the **misp-add-tag-to-event** command.
- Added the *is_local* argument to the **misp-add-tag-to-attribute** command.
```
